linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/tests/correct_value.pass.sh

```diff
@@ -5,7 +5,7 @@ source $SHARED/dconf_test_functions.sh
 install_dconf_and_gdm_if_needed
 
 clean_dconf_settings
-add_dconf_setting "org/gnome/login-screen" "banner-message-enable" "true" "gdm.d" "00-security-settings"
-add_dconf_lock "org/gnome/login-screen" "banner-message-enable" "gdm.d" "00-security-settings"
+add_dconf_setting "org/gnome/login-screen" "banner-message-enable" "true" "distro.d" "00-security-settings"
+add_dconf_lock "org/gnome/login-screen" "banner-message-enable" "distro.d" "00-security-settings"
```

Comment:
The changes in this test scenario make it only applicable for Fedora and RHEL9.

Is there a way to conditionally setup different gdm dconf directories?
Otherwise, this test should probably be duplicated, one copy for products using `gdm.d`, and another copy for products using `distro.d`.

Reply:
Did I suggest on one of my test system changes issues/PRs to use jinja for interpreting test cases prior to running them? It'd require reading product as well... But I think it might help to write platform-independent tests a little better (in addition to the test suite header)? Just a thought :)